components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/UIComponents/Drawer/Drawer.jsx

```diff
@@ -28,18 +29,6 @@ class Drawer extends Component {
     constructor() {
         super();
         this.closeDrawer = this.closeDrawer.bind(this);
-        this.scriptId = "drawer";
```

Comment:
Why this is removed?

Reply:
Those styles are in the default-theme.css.
drawer.css contains the layout of the drawer content.

Reply:
ok, then. +1